File: C#/Hashmap & Set/2215. Find the Difference of Two Arrays.cs
/* 2215. Find the Difference of Two Arrays
Difficulty: Easy
 https://leetcode.com/problems/find-the-difference-of-two-arrays/submissions/1385745357/ */

 public class Solution
{
    public IList<IList<int>> FindDifference(int[] nums1, int[] nums2)
    {
        var result = new List<IList<int>>();

        result.Add(new List<int>(nums1.Except(nums2)));
        result.Add(new List<int>(nums2.Except(nums1)));

        return result;
    }
}

File: Python/Hashmap & Set/2215. Find the Difference of Two Arrays.py
""" 2215. Find the Difference of Two Arrays
Difficulty: Easy
 https://leetcode.com/problems/find-the-difference-of-two-arrays/submissions/1385744538/ """

class Solution(object):
    def findDifference(self, nums1, nums2):
        a = set(nums1)
        b=  set(nums2)

        c = a-b
        d = b-a

        return [list(c), list(d)]
        